config/src/test/java/org/hyperledger/besu/config/JsonBftConfigOptionsTest.java

```diff
@@ -70,6 +71,13 @@ public void shouldGetDefaultBlockPeriodFromDefaultConfig() {
         .isEqualTo(EXPECTED_DEFAULT_BLOCK_PERIOD);
   }
 
+  @Test
+  public void shouldThrowOnNonPositiveBlockPeriod() {
+    final BftConfigOptions config = fromConfigOptions(singletonMap("BlockPeriodSeconds", -1));
```

Comment:
nit: could this be "blockperiodseconds" for consistency? Don't think lowercase matters here but it does in other config-related things so better to keep it lower everywhere I think.